package.json

```diff
@@ -29,6 +29,7 @@
     "big-integer": "^1.6.48",
     "flux": "^3.1.3",
     "gh-pages": "^3.1.0",
+    "moment": "^2.29.1",
```

Comment:
1. moment is already there so you don't have to explicitly add this.
2. Also please be sure to remove package-lock.json from this PR
3. it seems that you are using `npm` instead of `yarn`, please use yarn and commit yarn.lock whenever you change any dependency.

Reply:
Sure, I will do that

Reply:
FIXED